coops-ui/tests/unit/OfficeAddresses.spec.ts

```diff
@@ -0,0 +1,411 @@
+import Vue from 'vue'
```

Comment:
Need some extra tests in here to check when address are "same as ..." (different flavours of this).

Should also add check that "same as" is NOT displayed in other cases.

Reply:
Will do this!

Reply:
Awesome, thanks!